PROTO_tests/tests/logger_test.py

```diff
@@ -0,0 +1,100 @@
+import os
+import tempfile
+from logging import DEBUG, INFO, WARN, FileHandler, StreamHandler
+
+import pytest
+
+import arkouda as ak
+from arkouda import io_util
+from arkouda.logger import LogLevel, getArkoudaClientLogger, getArkoudaLogger
+
+
+class TestLogger:
+    logger_test_base_tmp = f"{os.getcwd()}/logger_io_test"
+    io_util.get_directory(logger_test_base_tmp)
+
+    def test_log_level(self):
+        assert "DEBUG" == LogLevel.DEBUG.value
+        assert "INFO" == LogLevel.INFO.value
+        assert "WARN" == LogLevel.WARN.value
+        assert "CRITICAL" == LogLevel.CRITICAL.value
+        assert "ERROR" == LogLevel.ERROR.value
+
+        assert LogLevel.DEBUG == LogLevel("DEBUG")
+        assert LogLevel.INFO == LogLevel("INFO")
+        assert LogLevel.WARN == LogLevel("WARN")
+        assert LogLevel.CRITICAL == LogLevel("CRITICAL")
+        assert LogLevel.ERROR == LogLevel("ERROR")
+
+    def test_arkouda_logger(self):
+        handler = StreamHandler()
+        handler.name = "streaming"
+        logger = getArkoudaLogger(name=self.__class__.__name__, handlers=[handler])
+        assert DEBUG == logger.level
+        assert "TestLogger" == logger.name
+        assert logger.getHandler("streaming") is not None
+        logger.debug("debug message")
+
+    def test_arkouda_client_logger(self):
+        logger = getArkoudaClientLogger(name="ClientLogger")
+        assert DEBUG == logger.level
+        assert "ClientLogger" == logger.name
+        logger.debug("debug message")
+        assert logger.getHandler("console-handler") is not None
+        with pytest.raises(ValueError):
+            logger.getHandler("console-handlers")
+
+    def test_update_arkouda_logger_log_level(self):
+        with tempfile.TemporaryDirectory(dir=TestLogger.logger_test_base_tmp) as tmp_dirname:
+            file_name = f"{tmp_dirname}/logger_test_output.txt"
+
+            logger = getArkoudaLogger(name="UpdateLogger")
+            assert DEBUG == logger.level
+            logger.debug("debug before level change")
+            logger.changeLogLevel(LogLevel.WARN)
+            assert WARN == logger.handlers[0].level
+            logger.debug("debug after level change")
```

Comment:
this section can be moved before the `with` block since it doesn't use the tempfile at all